packages/calling-component-bindings/src/videoGallerySelector.ts

```diff
@@ -94,8 +95,16 @@ export const videoGallerySelector: VideoGallerySelector = createSelector(
           renderElement: localVideoStream?.view?.target
         }
       },
-      remoteParticipants: _videoGalleryRemoteParticipantsMemo(remoteParticipants),
+      remoteParticipants: _videoGalleryRemoteParticipantsMemo(
+        updateUserDisplayNamesTrampoline(remoteParticipants ? Object.values(remoteParticipants) : [])
```

Comment:
nit: if this returns `[]` it will return a new array instance each time so isn't memozie - consider making a global const `noRemoteParticipants = []` to have a static ref object

Reply:
right will update this!